libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -17,37 +19,77 @@ describe('TabsComponent', () => {
     spectator = createHost({});
   });
 
-  describe('tabBarBottomHidden: true', () => {
-    it('should hide tab bar', () => {
-      spectator.component.tabBarBottomHidden = true;
-      spectator.detectChanges();
-      const ionTabBarElm = spectator.query('ion-tab-bar');
-
-      expect(ionTabBarElm).toHaveComputedStyle({ display: 'none' });
+  describe('tabBarBottomHidden', () => {
+    afterAll(() => {
+      TestHelper.resetTestWindow();
     });
-    it('should NOT set page footer safe area override if tab bar bottom is hidden', () => {
-      spectator.component.tabBarBottomHidden = true;
-      spectator.detectChanges();
-      const ionTabBarElm = spectator.query('ion-tab-bar');
 
-      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '' });
-    });
-  });
+    describe('screensize phone', () => {
+      beforeAll(async () => {
+        await TestHelper.resizeTestWindow(TestHelper.screensize.phone);
+      });
 
-  describe('tabBarBottomHidden: false', () => {
-    it('should show tab bar', () => {
-      spectator.component.tabBarBottomHidden = false;
-      spectator.detectChanges();
+      describe('tabBarBottomHidden: true', () => {
+        beforeEach(() => {
+          spectator.component.tabBarBottomHidden = true;
+          spectator.detectChanges();
+        });
+        it('should hide tab bar when tabBarBottomHidden', async () => {
```

Comment:
"when tabBarBottomHidden" is what? Isn't that already stated by the surrounding `describe` block?